File: MODUL2/SOAl10/PRAK110-2210817210016-AHMAD_REZA_ALFAYIET.py
import math

alas = 5; tinggi = 12
print ("Alas     =", alas, "cm")
print ("Tinggi   =", tinggi, "cm")

B = math.sqrt((pow(alas,2) + pow(tinggi,2)))
print ("Sisi A   =", tinggi, "cm")
print ("Sisi B   =", round(B), "cm")
print ("Sisi C   =", alas, "cm")
print ("Keliling =", round(alas + tinggi + B), "cm")
print ("Luas     =", round((alas * tinggi) / 2), "cm\u00B2")

File: MODUL2/SOAl10/PRAK110-2210817210016-AHMAD_REZA_ALFAYIET.c
#include <stdio.h>
#include <math.h>

int main()
{
    float alas = 5, tinggi = 12, B;
    printf ("Alas     = %.f cm\n", alas);
    printf ("Tinggi   = %.f cm\n", tinggi);

    B = sqrt((pow(alas,2) + pow(tinggi,2)));
    printf ("Sisi A   = %.f cm\n", tinggi);
    printf ("Sisi B   = %.f cm\n", B);
    printf ("Sisi C   = %.f cm\n", alas);
    printf ("Keliling = %.f cm\n", alas + tinggi + B);
    printf ("Luas     = %.f cm^2\n", (alas * tinggi) / 2);
}
